test/actions/create.spec.js

```diff
@@ -312,6 +312,26 @@ describe('Create action creator', () => {
       .to.not.throw('Item is missing in method argument and in config.body');
```

Comment:
I thought that it was somewhere in other parts of the code. I tried searching for anything that would resemble this exception but I didn't find anything. You're right, it seems to be obsolete.